styleguide/source/_patterns/01-atoms/forms/date-field.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This should likely not be included in this PR.

Reply:
@labbydev I already asked @toph and he said the default spacing should be fine

Reply:
@labbydev this PR helps the look and feel for the telephone PR and should be merged into develop before https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/192